indico/modules/events/contributions/templates/contrib_list_filter.html

```diff
@@ -40,7 +40,7 @@
             </div>
         </div>
         <div class="toolbar">
-            <a class="i-button js-reset-btn icon-close warning" type="button">
+            <a class="i-button js-reset-btn icon-close warning" type="button" href="#">
```

Comment:
`<a type=button>` makes no sense; can you remove that attribute? Also, I think `href=""` is enough - that way you also immediately notice when the default event is not being prevented (we never want `#` to show up in the URL, so with an empty href it'd reload the page ie be much more noticeable ;))

Reply:
Using `href=""` causes closing popup window after clicking this element. Maybe it should be replaced with `<button>` instead of `<a>`?

Reply:
If using `href=""` closes the popup window, are you sure `href="#"` doesn't append `#` to the URL? This sounds exactly like what I mentioned with the default not being prevented..

Reply:
I just checked and indeed, clicking the button adds a `#` to the URL. You need to add an evt preventDefault to the `$('.js-reset-btn').on('click', function() {` handler. Once that's done, `href="#"` vs `href=""` won't make a noticeable difference.

Reply:
However, the better option is to simply replace the `a` tag with a `button` tag. There's already `type="button"` on it so I think it's just an oversight.

Reply:
I updated this in master since there were a few more places with this issue. So when rebasing and likely getting a conflict you can simply remove this change here.